test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
@tmshort - the SessionTicket option is per-context (client/server). You'll need to extend the framework to accept a second server configuration; then you can configure tickets for each of server/second server/client separately. This should be entirely similar to how the first server is created, but if it looks like it's too much to handle, let me know and I'll do it for you.

Why is the "broken" option needed, do you want to test that callback failures are honoured? If that's important, then yes, you'll need to add an additional test configuration. See
https://github.com/openssl/openssl/commit/a263f320ebdb32ccc058ef02a617edbfe4a63e7f
for an example.